### PR TITLE
add the missing vmor.mm example

### PR DESCRIPTION
**Pull request by lhtin**

### CODE REVIEW

**Comment by aswaterman** [comment]
thanks!

File: v-spec.adoc [hunk 1]
```diff
@@ -4118,6 +4118,7 @@ possibly binary logical functions of the two input masks:
 | 0 | 0 | 0 | 1 | vmand.mm vd, src1, src2     |
 | 1 | 0 | 0 | 1 | vmxnor.mm vd, src1, src2    |
 | 0 | 1 | 0 | 1 | vmand.mm vd, src2, src2     | vmmv.m vd, src2
+| 0 | 1 | 1 | 1 | vmor.mm vd, src1, src2      |
 | 1 | 1 | 0 | 1 | vmorn.mm vd, src2, src1     |
 | 0 | 0 | 1 | 1 | vmand.mm vd, src1, src1     | vmmv.m vd, src1
 | 1 | 0 | 1 | 1 | vmorn.mm vd, src1, src2     |
```
Comment (David-Horner) [inline v-spec.adoc:4124]: wouldn't it be better added here, after 
| 1 | 0 | 1 | 1 | vmorn.mm vd, src1, src2 
?
following the binary sequence.

Comment (aswaterman) [reply v-spec.adoc:4124]: Yeah, I didn't realize the table was sorted.  I'll fix it.

Comment (aswaterman) [reply v-spec.adoc:4124]: 3ec63c12ae2bd4c68c3540afa6ab38ff442ed148

**Pull request merged**